<launch>

<node name="sink" pkg="network_monitor_udp" type="udpmonsink" args="12345" output="screen"/>
<node name="source" pkg="network_monitor_udp" type="udpmonsourcenode.py" output="screen"/>
<node name="test_node" pkg="network_monitor_udp" type="sample_bwtest.py" required="true" output="screen"/>

</launch> 


